### ISSUE TITLE
Test failure with Python 3.13

### ISSUE THREAD

**Issue by llimeht** [vmalloc/pyforge#27]
Hi! Our Debian Python maintainers have started testing packages with Python 3.13 and found that the pyforge test suite does not pass.

https://bugs.debian.org/1082257

The relevant section of the log is

```
=================================== FAILURES ===================================
___________ BinaryFunctionSignatureTest.test__binary_global_function ___________

self = <tests.test__signature_object.BinaryFunctionSignatureTest testMethod=test__binary_global_function>

    def test__binary_global_function(self):
        sig = FunctionSignature(time.time)
        self.assertEqual(sig._args, [])
>       self.assertTrue(sig.has_variable_args())
E       AssertionError: False is not true

tests/test__signature_object.py:132: AssertionError
```

(Note that there are other warnings about use of `setup.py`, some deprecated `setuptools` config and `pkg_resources` within the log file as other items to address in the future.)

**Comment by vmalloc** [comment]
@llimeht thanks for reporting! This is a relatively old repository that I don't get much time to maintain nowadays, but will try to get to it soon and see where the issue stems from

**Comment by llimeht** [comment]
Closed by #29
